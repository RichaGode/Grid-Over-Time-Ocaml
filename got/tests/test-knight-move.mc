int main()
{
  Grid g;
  Knight k;
  int x;
  int y; 
  int a;
  

  k = new_knight(30,20);
  g = grid_init(100,100);
  print(k.x);
  
  if (g.x>k.x){

    print(g.x);
    print(k.x);

    a = g.x - k.x;
    print(a);
    k = move_knight(k, g, 200, -10);
  }
  
  x = k.x;
  print(x);
  y = k.y;
  print(y); 

  return 0; 
}