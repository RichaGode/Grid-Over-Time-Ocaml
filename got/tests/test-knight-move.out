30
100
30
70
100
10